lib/irc-handler.js

```diff
@@ -141,6 +141,17 @@ var HELPERS = {
         context.logger.silly('Successfully sent bot message to user', data);
       }
     });
+  },
+
+  isNotMuted: function isNotMuted(context, mutedAction) {
+    if (context.irc &&
+        context.irc.hasOwnProperty('options') &&
+        context.irc.options &&
+        context.irc.options.hasOwnProperty('muteIrcEvents') &&
+        context.irc.options.mute.indexOf(mutedAction) !== -1) {
```

Comment:
This should obviously be `context.irc.options.muteIrcEvents.indexOf(...)`.

Reply:
Ah jeez, that was a slip on my part .-.